### PR TITLE
fix: ui refactoring of load more responses and comments button

### PR DESCRIPTION
**Pull request by muhammadadeeltajamul**
UI Refactoring 
- Corrected text for load more comments button
- Moved "Add a response" button below "Load more responses" button
- Change UI of "Load more responses", "Load more comments", "Add a response" and "Add a comment" button according to figma

Before
inf-547_before

After
inf-547_after

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -232,3 +232,7 @@ header {
   position: sticky;
   top: 0;
 }
+
+.color-00688d {
+  color: #00688D !important;
+}
```
Comment (awais-ansari) [inline src/index.scss:238]: This change is unnecessory because the link will have the same color code after applying the theme.

**Review by awais-ansari — CHANGES_REQUESTED** [review]

**Review by mehaknasir — APPROVED** [review]

File: src/discussions/comments/CommentsView.jsx [hunk 2]
```diff
@@ -80,14 +80,15 @@ function DiscussionCommentsView({
...
         {hasMorePages && !isLoading && (
           <Button
             onClick={handleLoadMoreResponses}
             variant="link"
             block="true"
-            className="card p-4"
+            className="card p-4 mb-4 color-00688d font-weight-500 font-size-14"
+            style={{
+              lineHeight: '20px',
+            }}
```
[the remark this answers is not in the document]
Comment (muhammadadeeltajamul) [reply src/discussions/comments/CommentsView.jsx:91]: Line height is needed. Without adding this property, the height of element decreases upto 1.333 px.

File: src/discussions/comments/comment/ResponseEditor.jsx [hunk 3]
```diff
@@ -37,7 +37,14 @@ function ResponseEditor({
...
       <div className={classNames({ 'mb-4': addWrappingDiv }, 'actions d-flex')}>
-        <Button variant="primary" className="px-2.5 py-2" onClick={() => setAddingResponse(true)}>
+        <Button
+          variant="primary"
+          className="px-2.5 py-2 font-size-14"
+          onClick={() => setAddingResponse(true)}
+          style={{
+            lineHeight: '20px',
+          }}
+        >
```
[the remark this answers is not in the document]
Comment (muhammadadeeltajamul) [reply src/discussions/comments/comment/ResponseEditor.jsx:47]: Total height of element decreases upto 1.333 px

File: src/discussions/comments/comment/Comment.jsx [hunk 4]
```diff
@@ -130,8 +130,11 @@ function Comment({
                 {(!isClosedPost && !inBlackoutDateRange(blackoutDateRange))
                   && (
                     <Button
-                      className="d-flex flex-grow mt-4.5"
+                      className="d-flex flex-grow mt-3 py-2 font-size-14"
                       variant="outline-primary"
+                      style={{
+                        lineHeight: '20px',
+                      }}
```
[the remark this answers is not in the document]
Comment (muhammadadeeltajamul) [reply src/discussions/comments/comment/Comment.jsx:137]: Total height of element decreases upto 1.333 px

**Review by awais-ansari — APPROVED** [review]

**Pull request merged**